cmake/AwsPrebuildDependency.cmake

```diff
@@ -39,11 +38,22 @@ function(aws_prebuild_dependency)
         list(APPEND cmakeCommand ${AWS_PREBUILD_CMAKE_ARGUMENTS})
     endif()
 
-    list(APPEND cmakeCommand WORKING_DIRECTORY ${depBinaryDir})
-    list(APPEND cmakeCommand RESULT_VARIABLE result)
+    # Set cmake prefix path via envrionment variable incase it's already a list.
+    # Some version of cmake will treat the values in the list as separate args.
+    if(WIN32)
+        list(JOIN CMAKE_PREFIX_PATH ";" PREFIX_PATH_ENV_VAR)
+    else()
+        list(JOIN CMAKE_PREFIX_PATH ":" PREFIX_PATH_ENV_VAR)
+    endif()
+    set(ENV{CMAKE_PREFIX_PATH} ${PREFIX_PATH_ENV_VAR})
```

Comment:
changing an environment variable seems like trouble
try to find a way to continue passing this as arguments
can do something to convince cmake more strongly that we want to treat this as a string, not a list